docs/src/modules/spring/pages/index.adoc

```diff
@@ -0,0 +1,22 @@
+= Developing with Spring (beta)
+
+include::ROOT:partial$include.adoc[]
+
+The Spring SDK provides a code-first approach to developing Kalix applications so that a developer does not need to worry about learning protobuf upfront.
+
```

Comment:
This PR is already merged, but by re-reading this paragraph, I have now a second thought. Should we even mention protobuf in the Spring SDK? 

We may want to keep it as such for know because in this initial phase, the words protobuf and gRPC is not completely hidden from the user. They will probably run into it in other parts of the documentation. 

But we may need to revisit it. Anyway, I just want to drop it here so we have a short discussion.

WDYT, @beritou?

Reply:
> I think this comes directly from the spring-sdk README

Yes it does.

> Should we even mention protobuf in the Spring SDK?

Nope!

> I would say more, even the term "code-first" if you think about it in isolation (without the context that there is some API-first counterparts) can be very surprising for someone who just lands on this page directly ("code-first as in are we supposed to skip design? or what does even mean?").

> So yeah, I think this might need some rewording in the future.

Great points and I completely agree. If you really think about it, this entire sentence can be removed.